tests/tensorflow2/test_keras.py

```diff
@@ -180,9 +181,10 @@ def test_keras_gradtape(out_dir, saveall):
 
     trial = smd.create_trial(path=out_dir)
     if saveall:  # save losses, metrics, weights, biases
-        assert len(trial.tensor_names()) == 10
+        assert len(trial.tensor_names()) == 15
         assert len(trial.tensor_names(collection=CollectionKeys.BIASES)) == 2
         assert len(trial.tensor_names(collection=CollectionKeys.WEIGHTS)) == 2
+        assert len(trial.tensor_names(collection=CollectionKeys.OPTIMIZER_VARIABLES)) == 5
```

Comment:
check vanilla TF as well

Reply:
Works if we add `hook.wrap_optimizer(opt)`